docs/topics/mta-cli-args.adoc

```diff
@@ -0,0 +1,248 @@
+|--version |Display the {ProductShortName} version.
+|--skipSourceCodeReports |This option skips generating a _Source code report_ when generating the application analysis report. Use this advanced option when concerned about source code information appearing in the application analysis.
+|====
+
+[id="cli-input-argument_{context}"]
+== Specifying the input
+
+A space-delimited list of the path to the file or directory containing one or more applications to be analyzed. This argument is required.
+
+.Usage
+[source,options="nowrap",subs="attributes+"]
+----
+--input <INPUT_ARCHIVE_OR_DIRECTORY> [...]
+----
+
+[id="cli-input-file-type-arguments_{context}"]
+== Analyze application source code
+
+Usage: `{mta-cli} analyze [flags]`
+
+The following flags are available:
+
+.{ProductShortName} Analyze application
+[cols="40%,60%a",options="header",]
+|====
+|Flag |Description
+
+| --analyze-known-libraries
+|  analyze known open-source libraries
+
+| -h, --help
+| help for analyze
+
+| -i, --input string
+|  path to application source code or a binary
+
+| --json-output
+| create analysis and dependency output as json
+
+| --list-sources
+| list rules for available migration sources
+
+| --list-targets
+| list rules for available migration targets
+
+| -l, --label-selector string
+| run rules based on specified label selector expression
+
+| --maven-settings string
+| path to a custom maven settings file to use
+
+| --overwrite
+| overwrite output directory
+
+| --skip-static-report
+|  do not generate static report
+
+| -m, --mode string
+| analysis mode" `full` (default) or `source-only`
+
+| -o, --output string | path to the directory for analysis output
+
+| --rules stringArray | filename or directory containing rule files
+
+| --skip-static-report
+| do not generate static report
+
+| -s, --source string
+| source technology to consider for analysis. To specify multiple sources, repeat the parameter: --source <source_1> --source <source_2> etc.
+
+| -t, --target string
+| target technology to consider for analysis. To specify multiple targets, repeat the parameter: --target <target_1> --target <target_2> etc.
+|====
+
+[id="cli-input-file-type-arguments-global-flags_{context}"]
+=== Global flags
+
+.{ProductShortName} Analyze application
+[cols="40%,60%a",options="header",]
+|====
+
+|Flag
+|Description
+
+| --log-level uint32
+| log level (default 4)
+
+| --no-cleanup
+| do not cleanup temporary resources
+|====
+
+
+
+[id="cli-output-argument_{context}"]
+== Specifying the output directory
+
+Specify the path to the directory to output the report information generated by {ProductShortName}.
+
+.Usage
+[source,options="nowrap",subs="attributes+"]
+----
+--output <OUTPUT_REPORT_DIRECTORY>
+----
+
+* If omitted, the report will be generated in an `<INPUT_ARCHIVE_OR_DIRECTORY>.report` directory.
```

Comment:
The `--output` argument can't be omitted anymore, if the user omits it, mta-cli will return an error:
 `Error: required flag(s) "output" not set
`

Reply:
Done